linalg/hypre.cpp

```diff
@@ -230,14 +239,13 @@ HypreParVector::~HypreParVector()
 
 void HypreParVector::ToNVector(N_Vector &nv)
 {
-   MFEM_ASSERT(nv && N_VGetVectorID(nv) == SUNDIALS_NVEC_PARHYP,
+   MFEM_ASSERT(nv && N_VGetVectorID(nv) == SUNDIALS_NVEC_PARALLEL,
                "invalid N_Vector");
```

Comment:
@gardner48, does this affect the current SUNDIALS integration?

Reply:
I believe this code was left over from a earlier version of the SUNDIALS interface. The parhyp vector is no longer used in interface so this can be removed.

Reply:
Ok, thanks for the quick reply

Reply:
I have removed the call to parhyp from `hypre.hpp` as suggested by @gardner48 .